composeApp/src/commonMain/kotlin/sp/bvantur/tasky/home/data/local/HomeLocalDataSource.kt

```diff
@@ -18,4 +25,10 @@ class HomeLocalDataSource(private val database: TaskyDatabase) {
     fun getEvents(): Flow<List<EventEntity>> = database.getEventDao().getAllEvents()
 
     fun getPendingAgendaItems(): Flow<List<EventEntity>> = database.getEventDao().getPendingEvents()
+
+    fun getProfileName(): String? = securePersistentStorageProvider.kVault.string(SecurePersistentStorageProvider.NAME)
```

Comment:
This looks to be an I/O operation. Calling it in a normal function will block the main thread more than we’d like. Best to make this function suspending.